content/blog/2020-pycon-india-mentorship.md

```diff
@@ -0,0 +1,25 @@
+Title: Want to perfect your proposal? The PyCon India mentorship team awaits you. 
+Date: 2020-07-27 00:34 
+Category: 2020 
+Slug: pycon-india-mentorship 
+Summary: Want to perfect your proposal? The PyCon India mentorship team awaits you.
+
+Do you have that one idea that can make the world “a better place”? (Silicon Valley Fans, Assemble!) 
+Do you wish to submit a proposal but cannot figure out the flow of your talk or how to present it? 
+We know it is not always easy to find the right words, but don’t you worry, the PyCon India Mentorship team is here just to help out with your proposal. 
+Whether this is your first time or you are a seasoned professional who wants someone to take a look at their work, the PyCon India Mentorship team is here just to help you out with your proposal. 
```

Comment:
The "PyCon India ....." section is the same as the previous line. 
Maybe try something like this
```suggestion
Whether this is your first time or you are a seasoned professional who wants someone to take a look at their work, our mentorship team will guide you through the process and help you wherever required. 
```